sonar-project.properties

```diff
@@ -1,5 +1,8 @@
 sonar.projectKey=ModernSoftwareDevelopement_taskusnow-api
 sonar.organization=modernsoftwaredevelopement
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.sourceEncoding=UTF-8
+sonar.exclusions=**/tests/unit_tests/*.ts,**/src/Task/routes/*.ts,**/src/*.ts
```

Comment:
line 5: I think the configuration should not have any knowledge of the business code (Task folder). Otherwise, when adding a new module, we would need to modify this configuration as well.

Reply:
Noted

> line 5: I think the configuration should not have any knowledge of the business code (Task folder). Otherwise, when adding a new module, we would need to modify this configuration as well.

Noted!

Reply:
But using the ** will impact all the folders instead of specific folders. anyway we will discuss more in the discussion.